templates/rhpam74-prod-immutable-kieserver-amq.yaml

```diff
@@ -497,6 +524,23 @@ objects:
     name: "${APPLICATION_NAME}-kieserver"
   roleRef:
     name: edit
+- kind: ServiceAccount
```

Comment:
where this service account is being used?
Can't we use the kieserver existent SVC?

Reply:
Not sure right now, probably copy pasted from the AMQ template. I can run the tests without the client if everything is ok.

Reply:
AMQ service account is needed for SSL configuration - see https://access.redhat.com/documentation/en-us/red_hat_amq/7.2/html-single/deploying_amq_broker_on_openshift_container_platform/index#generating-secret-ocp_broker-ocp